lasagne/layers/__init__.py

```diff
@@ -8,4 +8,5 @@
 from .shape import *
 from .merge import *
 from .normalization import *
+from .prelu import *
```

Comment:
`elemwise` was the first that came to my mind as well, but that could be confusing with the `ElemwiseSumLayer` in `layers.merge`, and a `NonlinearityLayer` with `softmax` is not exactly performing an element-wise operation. Besides, we have other layers performing element-wise operations in `layers.noise` and `layers.normalization`.

`transfer` and `activation` would be better in this respect, the only trouble is that we never called it like that anywhere else in the code. (But that was exactly the point, otherwise we'd call it `layers.nonlinearities`).)

What other layers would be added to this module in future? We probably don't want to rename it later, nor introduce yet another submodule just because the name doesn't fit.